l3kernel/testfiles/m3token002.lvt

```diff
@@ -263,6 +263,26 @@
 }
 \catcode`\!=12
 
+\TESTEXP { if_control_symbol }
```

Comment:
Add checks for `\if_true:`, `\if_false:`, `\else:`, and `\fi:` as well as active characters let to these to the tests!

Reply:
Add these checks for both new functions.